pkg/replicassyncer/controller.go

```diff
@@ -0,0 +1,102 @@
+package replicassyncer
```

Comment:
Can we put the current package under `pkg/controller`?

Reply:
Done.